### PR TITLE
WEBUI-251: remove options from pie chart since equal to default

### PR DESCRIPTION
**Pull request by andreacornaglia**
will also need to update charts-elements dependency version once nuxeo/chart-elements#4 is merged and a new version is generated.

### CODE REVIEW

**Review by mnixo — APPROVED** [review]

**Review by richardsd — APPROVED** [review]

File: elements/nuxeo-admin/nuxeo-repository-analytics.js [hunk 1]
```diff
@@ -184,12 +179,7 @@ Polymer({
       </nuxeo-repository-data>
 
       <nuxeo-card heading="[[i18n('repositoryAnalytics.topNCreators.heading', '10')]]">
-        <chart-pie
-          values="[[_values(topCreators)]]"
-          labels="[[_labels(topCreators)]]"
-          options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
-        >
-        </chart-pie>
+        <chart-pie values="[[_values(topCreators)]]" labels="[[_labels(topCreators)]]"> </chart-pie>
```
Comment (semisse) [inline elements/nuxeo-admin/nuxeo-repository-analytics.js:182]: nit: can we remove the extra blank space for consistency?
```suggestion
        <chart-pie values="[[_values(topCreators)]]" labels="[[_labels(topCreators)]]"></chart-pie>
```

File: elements/nuxeo-admin/nuxeo-repository-analytics.js [hunk 2]
```diff
@@ -243,12 +233,7 @@ Polymer({
       </nuxeo-repository-data>
 
       <nuxeo-card heading="[[i18n('repositoryAnalytics.filesByMimeType.heading')]]">
-        <chart-pie
-          values="[[_values(filesByMimeType)]]"
-          labels="[[_types(filesByMimeType)]]"
-          options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
-        >
-        </chart-pie>
+        <chart-pie values="[[_values(filesByMimeType)]]" labels="[[_types(filesByMimeType)]]"> </chart-pie>
```
Comment (semisse) [inline elements/nuxeo-admin/nuxeo-repository-analytics.js:236]: same here
```suggestion
        <chart-pie values="[[_values(filesByMimeType)]]" labels="[[_types(filesByMimeType)]]"></chart-pie>
```

**Review by semisse — APPROVED** [review]

**Comment by semisse** [comment]
I left some notes with nits, but 👍

**Review by richardsd — APPROVED** [review]
Looks good to me, please consider Sam's requests. 👍

File: elements/nuxeo-admin/nuxeo-repository-analytics.js (hunk 1 above)
Comment (andreacornaglia) [reply elements/nuxeo-admin/nuxeo-repository-analytics.js]: thanks! done!

File: elements/nuxeo-admin/nuxeo-repository-analytics.js (hunk 2 above)
Comment (andreacornaglia) [reply elements/nuxeo-admin/nuxeo-repository-analytics.js]: done

**Review by semisse — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**